about.html

```diff
@@ -1,121 +1,109 @@
         </p>
-      </div>
-      <img src="./images/creative-commons-license-logo.jpeg" alt="logo" />
+        <img class="blogo" src="img/techcrunchlogo.png" alt="Tech Crunch logo">
+    </section>
+    <section class="peventscontainer"><hr/>
+        <h2 class="peventshead">See past Microverse events</h2>
+        <p class="peventsdesc">Take a look at the revealing of the past three phases.</p>
+        <ul class="pevents">
+            <li class="pevent one">
+                <h3>2010</h3>
+                <p>Microverse 2018 in California</p>
+            </li>
+            <li class="pevent two">
+                <h3>2013</h3>
+                <p>Microverse 2019 in California</p>
+            </li>
+        </ul>
     </section>
 
-    <section class="about-page2">
-      <div class="prg1">
-        <h3 class="about-program">See the past CC Global Summits</h3>
-        <hr class="underline" />
-      </div>
-      <div class="about-images">
-        <div class="imageleft">
-          <img src="./images/image1.jpg" class="photo" alt="Confrence 2021" />
-        </div>
-        <div class="overlay">
-          <p class="overlay-text">Conference 2022</p>
-        </div>
-
-        <div class="imageleft">
-          <img src="./images/image2.jpg" class="photo" alt="Conference 2022" />
+    <footer class="footer">
+        <div class="partners">
+            <h2 class="partnerhead">Partner</h2>
+            <div class="partnerlogo">
+                <img class="plogo" src="img/tech.png" alt="Tech Crunch logo">
+                <img class="plogo" src="img/EdSurgelogo.png" alt="EdSurge logo">
+                <img class="plogo" src="img/CincoDíaslogo.png" alt="Cinco Días logo">
+                <img class="plogo" src="img/Infobaelogo.png" alt="Infobae logo">
+            </div>
         </div>
-        <div class="overlay">
-          <p class="overlay-text">Conference 2022</p>
+        <div class="bfooter">
+           
+                <img class="blogo" src="img/techcrunchlogo.png" alt="Tech Crunch logo">
+           
+            <div class="bfootertext">
+                <h4>2022 Microverse. Some rights reserved.</h4>
+                <p>Tech Crunch in association with Microverse.</p>
+            </div>
         </div>
-      </div>
-    </section>
-    <div class="footer-class2">
-      <div class="prg1">
-        <h3 class="program">Partners</h3>
-        <hr class="underline" />
-        <a class="bold">mozilla</a>
-        <a class="faint">Google</a>
-        <a class="bold">NAVER</a>
-        <a class="faint">daumkakao</a>
-        <a class="bold"><i class="fa fa-airbnb"></i>airbnb</a>
-      </div>
-    </div>
-
-    <script src="./style.js" async defer></script>
-  </body>
+    </footer>
```

Comment:
- [ ] Great job, here for the footer color, as specified in the design we should use the color **`#272131`**, which isn't fully black, kindly fix that.

![image](https://user-images.githubusercontent.com/59416136/169030078-6fbc50c6-55e2-43fc-8f3f-924ed1d670aa.png)